torch/quantization/fx/_equalize.py

```diff
@@ -347,6 +366,60 @@ def scale_weight_node(
     scaled_bias = torch.mul(bias, next_equalization_scale)
     modules[node.target].bias = nn.Parameter(scaled_bias)
 
+def scale_weight_functional(
+    op_node: Node,
+    model: GraphModule,
+    modules: Dict[str, nn.Module],
+    equalization_scale: torch.Tensor,
+) -> None:
+    """ Scales the weight value for functional layers
```

Comment:
would be good to add some comments for the tracebacks we are doing here.

Reply:
Ok, I will do so!